### ISSUE TITLE
php 5.6.10 - The requested URL returned error: 401 Unauthorized

### ISSUE THREAD

**Issue by craigcarnell** [Homebrew/homebrew-php#1783]
```
brew upgrade
==> Upgrading 4 outdated packages, with result:
memcached 1.4.24, homebrew/php/php56 5.6.10, homebrew/php/php56-mcrypt 5.6.10, homebrew/php/php56-opcache 5.6.10
Error: You must `brew link libevent' before memcached can be installed
==> Upgrading homebrew/php/php56
==> Downloading https://homebrew.bintray.com/bottles-php/php56-5.6.10.yosemite.bottle.tar.gz

curl: (22) The requested URL returned error: 401 Unauthorized
Error: Failed to download resource "php56"
Download failed: https://homebrew.bintray.com/bottles-php/php56-5.6.10.yosemite.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror

curl: (60) SSL certificate problem: Invalid certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
Error: Failed to download resource "php56"
Download failed: https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror
==> Upgrading homebrew/php/php56-mcrypt
==> Upgrading homebrew/php/php56-opcache
```

**Comment by alanthing** [comment]
I can't speak for the failure on https://www.php.net, but I think you caught the bottle URL being unavailable during a very brief time as the bottles were being published. For example, I'm able to load https://homebrew.bintray.com/bottles-php/php56-5.6.10.yosemite.bottle.tar.gz, copied and pasted from your issue, in a new browser window and it loads anonymously from BinTray. Give it another shot.

**Comment by craigcarnell** [comment]
@alanthing Thanks! latest error is:

```
brew upgrade
==> Upgrading 3 outdated packages, with result:
memcached 1.4.24, homebrew/php/php56-mcrypt 5.6.10, homebrew/php/php56-opcache 5.6.10
Error: You must `brew link libevent' before memcached can be installed
==> Upgrading homebrew/php/php56-mcrypt
==> Downloading https://homebrew.bintray.com/bottles-php/php56-mcrypt-5.6.10.yosemite.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "php56-mcrypt"
Download failed: https://homebrew.bintray.com/bottles-php/php56-mcrypt-5.6.10.yosemite.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror

curl: (60) SSL certificate problem: Invalid certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
Error: Failed to download resource "php56-mcrypt"
Download failed: https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror
==> Upgrading homebrew/php/php56-opcache
==> Downloading https://homebrew.bintray.com/bottles-php/php56-opcache-5.6.10.yosemite.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "php56-opcache"
Download failed: https://homebrew.bintray.com/bottles-php/php56-opcache-5.6.10.yosemite.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror

curl: (60) SSL certificate problem: Invalid certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
Error: Failed to download resource "php56-opcache"
Download failed: https://www.php.net/get/php-5.6.10.tar.bz2/from/this/mirror
```

**Comment by craigcarnell** [comment]
@alanthing The annoying thing is I'm only trying to install php56-intl which insisted on downloading php 5.6.10 :)

**Comment by Stiffi** [comment]
I got the same error

**Comment by marcosbrasil** [comment]
I got the same error... url of php.net must be "https://php.net ..." without "www"

works with me ;)

**Comment by craigcarnell** [comment]
@marcosbrasil How exactly do I do that? Thanks :)

**Comment by craigcarnell** [comment]
@marcosbrasil I figured it out!

**Comment by marcosbrasil** [comment]
`brew edit php56` and I changed url constant... But someone may alter this and commit to homebrew-php...

**Comment by KvanSteijn** [comment]
I have the same error, but i can't found any link in the code. I used the command brew edit php56.

**Comment by DsXack** [comment]
@KvanSteijn, you must replace `PHP_SRC_TARBALL` to `https://php.net/get/php-5.6.10.tar.bz2/from/this/mirror`, or wait for solve problem

**Comment by alanthing** [comment]
Resolved in #1786.
